README.md

```diff
@@ -230,7 +230,7 @@ python -m ai_feedback \
   --assignment <assignment_directory> \
   --question <question_number> \
   --model <model_name> \
-  --output <markdown|stdout|direct>
+  --output <markdown|direct>
```

Comment:
This change isn't technically correct: the user still can pass `stdout` as an option, it's just no longer necessary to do so.